Makefile

```diff
@@ -32,6 +32,12 @@ test:
 test-api:
 	py.test $(PYTEST_OPTIONS) integrade/tests/api/v1
 
+test-ui:
+	py.test $(PYTEST_OPTIONS) integrade/tests/ui --driver=Firefox
```

Comment:
Any particular reason to prefer Firefox? (I am not saying that we should use any particular browser, I'm just curious why you chosen Firefox :smile:)

Reply:
I just hadn't installed Chrome yet, actually. I ran it on Chrome later once all the tests were in place, but Firefox > Chrome for my personal taste so I already had it!